rpm/usbutils.spec

```diff
@@ -19,44 +15,22 @@ BuildRequires:  libtool
 This package contains utilities for inspecting devices connected to a
 USB bus.
 
-%package devel
-Summary:    Development files for %{name}
-Group:      Development/Libraries
-Requires:   %{name} = %{version}-%{release}
-
-%description devel
-Development files for %{name}.
-
-
 %prep
-%setup -q -n %{name}-%{version}/%{name}
-
-# usbutils-006-hwdata.patch
-%patch0 -p1
-# usbutils-make-hwdata.patch
-%patch1 -p1
+%autosetup -p1 -n %{name}-%{version}/%{name}
 
 %build
 ./autogen.sh
 %configure --disable-static \
     --sbindir=%{_sbindir} \
-    --datadir=%{_datadir}/hwdata \
-    --disable-usbids
+    --datadir=%{_datadir}/hwdata
 
-make %{?jobs:-j%jobs}
+%make_build
 
 %install
-rm -rf %{buildroot}
 %make_install
 
 %files
 %defattr(-,root,root,-)
-%doc COPYING
+%license LICENSES/GPL-2.0-only.txt
 %{_bindir}/*
-
-%files devel
-%defattr(-,root,root,-)
-%doc AUTHORS COPYING ChangeLog NEWS README
-%{_datadir}/pkgconfig/usbutils.pc
-%{_mandir}/*/*
-
+%exclude %{_mandir}/*/*
```

Comment:
That could go to a doc subpackage.
See JB#55515.

Reply:
That's mostly about api documentation, html and such. Man pages on mobile devices are not really that useful.

Reply:
This would have been for SDK not for device.

Reply:
Then again usb stuff on sdk, hm.